web-client/components/ViewWrapper.js

```diff
@@ -1,4 +1,4 @@
-function ViewWrapper(props) {
+const ViewWrapper = props => {
```

Comment:
Depending on what we end up doing with the stying, this could maybe be turned into a reusable styled component instead.

Reply:
agreed